example/cmd/simple-device/main.go

```diff
@@ -0,0 +1,19 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2017-2018 Canonical Ltd
+// Copyright (C) 2018 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// This package provides a simple example of a device service.
+package main
+
+import (
+	"github.com/edgexfoundry/device-sdk-go/example/driver"
+	"github.com/edgexfoundry/device-sdk-go/startup"
+)
+
+func main() {
+	sd := driver.SimpleDriver{}
+	startup.Bootstrap(&sd)
```

Comment:
I don't this approach as it removes flexibility. What if the developer wants to define additional comman line parameters? What if they'd like to wire up their own POSIX signal handlers?

Reply:
I've made startup package optional, so user could choose use startup package or on their way to start the service